### PR TITLE
fix clang warnings, directly include wtypes.h

### PR DESCRIPTION
**Pull request by bmiklautz**
* Directly include winpr/wtypes.h where _fseeki64 or _ftelli64 is used.
* fix clang 5 warnings.

### CODE REVIEW

File: libfreerdp/codec/bitmap.c [hunk 1]
```diff
@@ -435,8 +435,8 @@
 #define TEST_MIX \
 		((last_line == 0 && pixel == mix) || \
 				(last_line != 0 && pixel == (ypixel ^ mix)))
-#define TEST_FOM (TEST_FILL || TEST_MIX)
-#define TEST_COLOR (pixel == last_pixel)
+#define TEST_FOM TEST_FILL || TEST_MIX
```
Comment (hardening) [inline libfreerdp/codec/bitmap.c]: I don't see why it's a gain to have all these added parenthesis below...

**Comment by bmiklautz** [comment]
@freerdp-bot test.

File: libfreerdp/codec/bitmap.c (hunk 1 above)
Comment (bmiklautz) [inline libfreerdp/codec/bitmap.c]: In fact it's not really a gain but with -Wparentheses-equality there are some warnings but I think it's worth to have the warnings enabled in general.

**Comment by bmiklautz** [comment]
@freerdp-bot test test test.

**Pull request merged**